libs/wifi---esp32/ca-bundle.crt

```diff
@@ -0,0 +1,3191 @@
+##
```

Comment:
do you have more docs about this file, like how to generate it and so forth?

Reply:
added instructions